news/3257.feature

```diff
@@ -0,0 +1,2 @@
+Add IMREALLYPLONE6 Python marker
```

Comment:
Can you fix the news snippet to use the new `PLONE60MARKER`? After that, it is fine to merge it yourself.
Will you create a PR to add the `PLONE52MARKER` in 5.2?